### PR TITLE
Change IDD to Products dir

### LINKED ISSUE #4724: Packaging fails to find idd
The CLI change of the idd location seems to have broken ```make package```.

CMake Error at /home/elee/EnergyPlus/GitHub/EnergyPlusTeamBranch2/builds/doctools/d/cmake_install.cmake:40 (file):
  file INSTALL cannot find
  ".../builds/doctools/d/Energy+.idd".

Just need to add Products/ in there.

@nealkruis @kbenne @lefticus

### PR DESCRIPTION
**Pull request by Myoldmopar**
Fixes #4724

### CODE REVIEW

File: cmake/RunSimulation.cmake [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 get_filename_component(IDF_NAME "${IDF_FILE}" NAME_WE)
 get_filename_component(IDF_EXT "${IDF_FILE}" EXT)
-get_filename_component(EXE_PATH "${ENERGYPLUS_EXE}" PATH)
+get_filename_component(EXE_PATH "${ENERGYPLUS_EXE}" DIRECTORY)
```
Comment (kbenne) [inline cmake/RunSimulation.cmake]: PATH was deprecated in favor of DIRECTORY in cmake 3.

**Comment by kbenne** [comment]
@nealkruis @Myoldmopar @lefticus @mjwitte @mbadams5 I need to build some consensus on update to cmake v3 or I need to work around the requirement.  Please weigh in.  If I merge this (after getting some green lights) your build will not work unless we force everyone to update.  

I think the main issue is that v3 has is not in the Ubuntu 14.04 package manager.  Windows and Mac people are probably just running the official installer packages so it probably isn't as jarring.  But still everyone will have to update if they are < 3.

**Comment by Myoldmopar** [comment]
I'm _resistant_ to the push to cmake 3 just for this change.  But I'm not _against_ it.  Things were working really well up until now.  Just let me know so I can upgrade cmake on all the Linux boxes.

**Comment by lefticus** [comment]
I'm *strongly* resistant. But I don't believe I have any veto power on this project... :)

**Comment by kbenne** [comment]
Thats two against.  Plus I'm not even sure I'm in favor when I realized the package manager doesn't have it.  I'm going to commit a workaround that only requires 2.8.

**Comment by mjwitte** [comment]
@kbenne Please be sure to give everyone a heads-up when this gets merged in.  If I'm following correctly, this will put the built IDD into ...Build\Products\Release or ...Build\Products\Debug correct?
Please also update the Windows section of the readme, Step 9 for Visual Studio to set the Working Directory to ($TargetDir) or add a Command Argument to point to the idd.  Readme should also explain that input/output files will be in the Working Directory.  I'm assuming cmake can't set these properties directly.

**Comment by kbenne** [comment]
@mjwitte Roger that.  I will update docs and notify vie developer email.  At this point I'm still working though those test failures which are saying it can't find the idd.  What?  ughhh!  works for me locally.

**Comment by kbenne** [comment]
Not going to merge this.  Going with https://github.com/NREL/EnergyPlus/pull/4755 instead.

**Pull request closed without merging**